### ISSUE TITLE
alt texts in publisher epubs

### ISSUE THREAD

**Issue by TorilBWM** [nlbdev/nordic-accessible-epub-guidelines#54]
Some publisher epubs contain image descriptions as alt texts. Should we specify that this text should be preserved? See also #24.

**Comment by jonaslil** [comment]
Good point. It seems clear that they need to be preserved.
